specification/signalr/resource-manager/Microsoft.SignalRService/stable/2022-02-01/examples/SignalRCustomDomains_CreateOrUpdate.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "body": {
+      "properties": {
+        "certificateName": "cert"
+      }
+    },
+    "api-version": "2022-02-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "resourceName": "mySignalRService",
+    "domainName": "example.com"
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "properties": {
+          "provisioningState": "Succeeded",
+          "certificateName": "cert"
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myResourceGroup/providers/Microsoft.SignalRService/SignalR/mySignalRService/customDomain/example.com",
+        "name": "example.com"
```

Comment:
if you are using the resource name as the indication of the domain name you will only ever be able to support domain names that are valid resource names (https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#arguments-for-crud-on-resource).  FYI as this may impact you down the road

Reply:
Nice catch. Added a dedicated property `domainName`.